ibm_catalog.json

```diff
@@ -0,0 +1,183 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-container-registry",
+      "label": "IBM Container Registry",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "container",
```

Comment:
As per `ibmcloud catalog filter options --all` the valid tag name is `containers`

Reply:
addressed